app/src/main/res/layout/sos_setting_field.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
```

Comment:
Each parameter should be on a new line. Also, I don't see why we would need linear layout here.